apps/frontend/src/pages/quiz/quiz.tsx

```diff
@@ -30,6 +30,12 @@ const Quiz: React.FC = () => {
 		hasAnsweredOnboardingQuestions: auth.user?.hasAnsweredOnboardingQuestions,
 	}));
 
+	useEffect(() => {
```

Comment:
This is already handled on my task for retaking quiz

Reply:
Also needed it here since after sending onboarding it refreshes the state. So if it refreshes the page it will always be onboarding screen.